spec/paperclip/storage/s3_live_spec.rb

```diff
@@ -152,8 +154,8 @@
                       path: ":class/:attachment/:id/:style.:extension",
                       s3_region: ENV["S3_REGION"],
                       s3_credentials: {
-                        aws_access_key_id: ENV['AWS_ACCESS_KEY_ID'],
-                        aws_secre_access_key: ENV['AWS_SECRET_ACCESS_KEY']
+                        access_key_id: ENV['AWS_ACCESS_KEY_ID'],
+                        secret_access_key: ENV['AWS_SECRET_ACCESS_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Put a comma after the last item of a multiline hash.